Dockerfile

```diff
@@ -31,10 +31,14 @@ RUN apk --update --no-cache add \
       boto==2.49.0 \
       boto3==1.21.1 \
       iteration-utilities==0.11.0 \
-      pre-commit \
       PyGithub==1.55 && \
     git config --global advice.detachedHead false
 
+## Workaround https://github.com/pypa/pip/issues/5247
+RUN pip3 install --upgrade --force-reinstall pip==9.0.3 && \
+    pip3 install --no-cache-dir --disable-pip-version-check pre-commit && \
+    pip3 install --upgrade pip
+
```

Comment:
Why is this needed? The referenced issue is for Pip 9 and 10 and Python 2.7. We are using Pip v22 and Python 3.9. Any problem this is fixing should probably be fixed some other way.

Reply:
But the problem was still there 
https://github.com/cloudposse/build-harness/runs/7847715248?check_suite_focus=true

```
docutils, colorama, cfgv, virtualenv, pynacl, deprecated, botocore, s3transfer, PyGithub, pre-commit, boto3, awscli
#5 32.27   Attempting uninstall: distlib
#5 32.27     Found existing installation: distlib 0.3.3
#5 32.27 ERROR: Cannot uninstall 'distlib'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```

Reply:
@Nuru ^